lang/cs/Org.Apache.REEF.Wake/Remote/IConnectionRetryHandler.cs

```diff
@@ -15,7 +15,12 @@
 // specific language governing permissions and limitations
 // under the License.
 
+#if NOT_DOTNET_BUILD
```

Comment:
Aren't there constants defined as part of the new build system that allow us to switch based on the target?